charts/rancher-gke-operator/1.0.000/templates/clusterrole.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: gke-operator
+  namespace: cattle-system
+rules:
```

Comment:
No secrets in here yet, because no cloud creds yet ? We'll need to remember that.

Reply:
I do think secrets are being read at least already, right?

Reply:
Not currently but they will be shortly, I can add that back since we know it will be needed